Decred Wallet/Features/Transactions/TransactionTableViewCell.swift

```diff
@@ -12,84 +12,95 @@ import Dcrlibwallet
 class TransactionTableViewCell: UITableViewCell {
     @IBOutlet weak var dataImage: UIImageView!
     @IBOutlet weak var dataText: UILabel!
+    @IBOutlet weak var secondaryDataLabel: UILabel!
+    @IBOutlet weak var voteInfoLabel: Label!
     @IBOutlet weak var status: UILabel!
-   // @IBOutlet weak var dateT: UILabel!
+    @IBOutlet weak var secondaryStatusLabel: UILabel!
     @IBOutlet weak var statusIcon: UIImageView!
-    
+
     var count = 0
-    
+
     override func awakeFromNib() {}
-    
+
     override class func height() -> CGFloat {
         return 56
     }
-    
+
     func setData(_ data: Any?) {
         if let transaction = data as? Transaction {
             var confirmations: Int32 = 0
             if transaction.blockHeight != -1 {
                 confirmations = WalletLoader.shared.firstWallet!.getBestBlock() - Int32(transaction.blockHeight) + 1
             }
-            let Date2 = NSDate.init(timeIntervalSince1970: TimeInterval(transaction.timestamp) )
-                      let dateformater = DateFormatter()
-                      dateformater.locale = Locale(identifier: "en_US_POSIX")
-                      dateformater.dateFormat = "MMM dd"
-                      dateformater.string(from: Date2 as Date)
-                      
-                     // self.dateT.text = dateformater.string(from: Date2 as Date)
-            
+
+            let txDate = NSDate.init(timeIntervalSince1970: TimeInterval(transaction.timestamp))
+            let dateformater = DateFormatter()
+            dateformater.locale = Locale(identifier: "en_US_POSIX")
+            dateformater.dateFormat = "MMM dd"
+            dateformater.string(from: txDate as Date)
+            let ageInDays = (txDate as Date).daysFromNow
+            var dateString = dateformater.string(from: txDate as Date)
+
+            if ageInDays == 0 {
+                dateString = LocalizedStrings.today
+            } else if ageInDays == -1 {
+                dateString = LocalizedStrings.yesterday
+            }
```

Comment:
Can be refactored thus:
```
let txDate = Date(timeIntervalSince1970: TimeInterval(transaction.timestamp))
let ageInDays = txDate.daysFromNow
var txDateString: String

if ageInDays == 0 {
    txDateString = LocalizedStrings.today
} else if ageInDays == -1 {
    txDateString = LocalizedStrings.yesterday
} else {
    let txDateIsInCurrentYear = Calendar.current.isDate(txDate, equalTo: Date(), toGranularity: .year)
    txDateString = txDate.toString(format: txDateIsInCurrentYear ? "MMM dd" : "MMM dd, YYYY")
}
```

Above code depends on adding an extension `toString()` function to the `Date` class:
```
func toString(format: String, localeIdentifier: String = "en_US_POSIX") -> String {
    let dateformater = DateFormatter()
    dateformater.locale = Locale(identifier: localeIdentifier)
    dateformater.dateFormat = format
    return dateformater.string(from: self)
}
```